### ISSUE TITLE
Sphix not in setup.py

### ISSUE THREAD

**Issue by nsorros** [wellcometrust/WellcomeML#101]
`make dist` which depends on `make update-docs` fails saying
```
build/virtualenv/bin/sphinx-apidoc --no-toc -d 5 -H WellcomeML -o ./docs -f wellcomeml
make: build/virtualenv/bin/sphinx-apidoc: No such file or directory
```

possible fix `pip install sphinx`

**Comment by nsorros** [comment]
After installing sphinx with `pip install sphinx` i got 
```
Could not import extension sphinx_rtd_theme (exception: No module named 'sphinx_rtd_theme')
```

**Comment by aCampello** [comment]
Ah damn, this was probably caused by removing the requirements.txt!

**Comment by aCampello** [comment]
Oh no, there is a docs requirements. The `make dist` command was developed before the docs. We need to `pip install -r docs/requirements.txt`

**Comment by nsorros** [comment]
Fixes by 4f0d495d5393d9686afd2551067b32c096bbab46 but we can also go with your suggested fix which probably makes more sense

**Comment by nsorros** [comment]
Hmm, on second though this problem should not have occured due to https://github.com/wellcometrust/WellcomeML/blob/3a8097b8c4c27ce4e4abfb62f488ce9041a8fbb3/Makefile#L11

So maybe it happened because i had not recreated the environment. Let me try this.
